### DISCUSSION TITLE
Geopotential height as output in P7, P8?

### DISCUSSION THREAD

**Discussion by benjamin-cash** [ufs-community/ufs-weather-model#1238] [Q&A] [answered]
Is there a way to get geopotential height as a model output field in P7 and P8? I used to have it output directly on pressure levels via the atmos_4x fields, but I understand that has been deprecated. Can it be included directly in the diag_table for output on model levels?

**Comment by benjamin-cash** [comment, marked as answer]
Apparently this requires UPP.

**Comment by yangfanglin** [comment]
HGT is included in the P7/P8 grib2 outputs,   at all resolutions,  on standard isobaric layers.  It has always been included in GFS.v15/v16 outputs as well.

**Reply by benjamin-cash** [reply]
HI @yangfanglin - Thanks! Is it just not available in the netcdf output then? Or is there some way to add it into those files?

**Comment by yangfanglin** [comment]
@benjamin-cash In the history netcdf file you should be able to find delta-z (delz) on the model native vertical coordinate.

**Reply by benjamin-cash** [reply]
That's what I'm using to calculate it now, I just thought it might be available directly.
